packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -50,14 +50,22 @@
 }
 
 .wp-block-navigation-menu-item {
-	font-family: $editor-font;
-	font-weight: bold;
-	font-size: $text-editor-font-size;
-	background-color: var(--background-color-menu-link);
-	color: var(--color-menu-link);
+	&:not(.is-editing) {
+		font-family: $editor-font;
+		font-weight: bold;
+		font-size: $text-editor-font-size;
+		background-color: var(--background-color-menu-link);
```

Comment:
I'm afraid we can't use CSS variables because they're not supported in IE. We'll have to set these as inline styles with JS instead.